packages/http/test/http-decorators.test.ts

```diff
@@ -271,6 +271,14 @@ describe("http: decorators", () => {
           code: "@typespec/http/shared-inconsistency",
           message: `All shared routes must agree on the value of the shared parameter.`,
         },
+        {
+          code: "@typespec/http/shared-inconsistency",
+          message: `All shared routes must agree on the value of the shared parameter.`,
+        },
+        {
+          code: "@typespec/http/shared-inconsistency",
+          message: `All shared routes must agree on the value of the shared parameter.`,
```

Comment:
Can we be more specific about what the shared parameter is?

Reply:
sorry I didn't update this message with the new one, test are failing